### PR TITLE
EVG-14328 use project identifier for patch workflow

### PR DESCRIPTION
**Pull request by ablack12**
See https://github.com/evergreen-ci/evergreen/pull/4567 for details.

### CODE REVIEW

File: src/pages/task/Metadata.tsx [hunk 1]
```diff
@@ -61,15 +61,14 @@ export const Metadata: React.FC<Props> = ({ loading, task, error, taskId }) => {
...
   const { baseTaskDuration, baseTaskLink } = baseTaskMetadata ?? {};
-
+  const projectIdentifier = task?.project?.identifier;
   const { author, patchID } = patchMetadata ?? {};
   const oomTracker = details?.oomTracker;
 
   const hostLink = getHostRoute(hostId);
   const distroLink = `${getUiUrl()}/distros##${distroId}`;
-
+  console.log("project identifier: ", projectIdentifier);
```
Comment (khelif96) [inline src/pages/task/Metadata.tsx:71]: left over console.log

File: src/pages/task/Metadata.tsx [hunk 2]
```diff
@@ -61,15 +61,14 @@ export const Metadata: React.FC<Props> = ({ loading, task, error, taskId }) => {
   } = task || {};
 
   const baseCommit = revision?.slice(0, 10);
-
   const { baseTaskDuration, baseTaskLink } = baseTaskMetadata ?? {};
-
+  const projectIdentifier = task?.project?.identifier;
```
Comment (khelif96) [inline src/pages/task/Metadata.tsx:65]: you should destructure this field above like the other task fields

File: src/pages/patch/Metadata.tsx [hunk 3]
```diff
@@ -29,7 +30,15 @@ export const Metadata: React.FC<Props> = ({ loading, patch, error }) => {
   const { makespan, timeTaken } = duration || {};
   return (
     <MetadataCard loading={loading} error={error} title="Patch Metadata">
-      <P2>Project Name: {projectID}</P2>
+      <P2 data-cy="task-metadata-project">
```
Comment (khelif96) [inline src/pages/patch/Metadata.tsx:33]: If you aren't using this data-cy in any tests it can be omitted

File: src/pages/patch/Metadata.tsx [hunk 4]
```diff
@@ -29,7 +30,15 @@ export const Metadata: React.FC<Props> = ({ loading, patch, error }) => {
...
+        Project:{" "}
+        <StyledLink
+          data-cy="project-link"
```
Comment (khelif96) [inline src/pages/patch/Metadata.tsx:36]: same with this data-cy

**Comment by ablack12** [comment]
evergreen retry

**Review by SupaJoon — APPROVED** [review]

**Review by khelif96 — APPROVED** [review]

**Pull request merged**